docs/configuration/logging.md

```diff
@@ -0,0 +1,10 @@
+## Logging
```

Comment:
can you link this from the docs/README?

Reply:
There was already a link. That's why I chose this location for the file 😄